examples/requests/plane/main.js

```diff
@@ -0,0 +1,19 @@
+Core.processGlobal();
+Plane.askPermission();
+
+if (!Plane.started) {
```

Comment:
@myfoxtail I've added waitForStart method to plane. I've done it because in this example I wanted to show how successful and failed requests works. So, I don't want to make one request and wait for it. I want to make many of them and ping dispatcher every time. It's just for training purposes (for user)